### ISSUE TITLE
Parallel has issues.

### ISSUE THREAD

**Issue by nutant233** [GregTechCEu/GregTechCEu-1.19#1452]
### Checked for existing issues

- [X] I have checked for existing issues, and have found none.

### Tested latest version

- [X] I have checked that this occurs on the latest version.

### GregTech CEu Version

1.3.0-build_811

### Recipe Viewer Installed

JEI

### Environment

Singleplaye

### Cross-Mod Interaction

Yes

### Other Installed Mods

jei

### Expected Behavior

The machine is working normally in parallel.

### Actual Behavior

Machines do not work unless there is no parallel number.

### Steps to Reproduce

https://github.com/GregTechCEu/GregTech-Modern/assets/65939836/b8ed9e99-878e-4562-9ea9-7113d2fdf66a

**Comment by screret** [comment]
duplicate of #1448.
